src/app/app-routing.module.ts

```diff
@@ -4,58 +4,74 @@ import {MainPage} from './shared/enums/main-page.enum';
 import {NotFoundErrorPageComponent} from './error-handling/components/not-found-error-page/not-found-error-page.component';
 import {FatalErrorPageComponent} from './error-handling/components/fatal-error-page/fatal-error-page.component';
 import {fatalErrorMapGuard} from './embedded-page/guards/fatal-error-page.guard';
+import {environment} from '../environments/environment';
 
 const siteOperator = 'Geoportal Kanton Zürich';
+const createRouteTitle = (title?: string): string => {
+  let routeTitle = '';
+  if (title) {
+    routeTitle = `${title} `;
+  }
```

Comment:
I think we were both a bit tired yesterday :D This can be simplified to - you can accept it directly here on Github if you agree.
```suggestion
  const routeTitle = title ? `${title} ` : '';
```